group_vars/rabbitmq/public.yml

```diff
@@ -6,4 +6,4 @@ SANITY_CHECK_LIVE_PORTS:
     port: 15671
     message: "RabbitMQ management does not respond on port 15671."
 
-TARSNAP_BACKUP_FOLDERS: /etc/rabbitmq /var/lib/rabbitmq /etc/letsencrypt
+TARSNAP_BACKUP_FOLDERS: /usr/local/lib/rabbitmq/backup /etc/rabbitmq /etc/letsencrypt
```

Comment:
I think we can simply use `{{ RABBITMQ_EXPORT_DIR }}` as the first directory here, to guarantee it matches.